README.md

```diff
@@ -123,15 +123,15 @@ correct Airflow tag/version/branch and python versions in the URL.
 1. Installing just Airflow:
 
 ```bash
-pip install apache-airflow==1.10.12 \
- --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-1.10.12/constraints-3.7.txt"
+pip install apache-airflow==1.10.13 \
```

Comment:
shall line 117 in README to be updated as well? Similar to the change done in `docs/installation.rst`

Reply:
In addition, in `docs/production-deployment.rst`, there are three sample Dockerfiles are using `FROM apache/airflow:1.10.12`. Should they be updated as well?

Reply:
On phone so not adding comments at the exact code lines. Sorry for that

Reply:
L117 says that the change was introduced in 1.10.12 :) hence I have not changed that.

Updated the dockerfiles in `docs/production-deployment.rst` to use 1.10.13 in e6db852